libqpdf/JSON.cc

```diff
@@ -410,6 +410,16 @@ JSON::isNull() const
     return m->value->type_code == vt_null;
 }
 
+JSON
+JSON::getDictItem(std::string const& key) const
+{
+    if (auto v = dynamic_cast<JSON_dictionary const*>(m->value.get())) {
+        if (auto it = v->members.find(key); it != v->members.end())
```

Comment:
Please put braces around the code in the condition.

Reply:
Done. Also a couple of tests added.